### ISSUE TITLE
Closing nicehash when pressing "start"

### ISSUE THREAD

**Issue by YoWohoKid** [nicehash/NiceHashMiner-Archived#1176]
I am new to nicehash. Whenever I press "start" it just crashes or closes. I get no errors or what so ever. It just dissepears. What could be an issue?

**Comment by OneOfMany07** [comment]
Yeah, I saw the same thing at first.  I ended up re-downloading and playing around with settings.  I can't say if it was my changes that made things work or just waiting (maybe a server came up?), but eventually I'm running now.

Might try an older version.  Or changing the Service location.

Also I had issues from my antivirus program quarantining the files the app downloaded for each crypto mining process.  I had to whitelist the folder.  Then release each file again.  BitDefender.  Apparently very good protection, even from not really viruses.  I submitted a bunch of the files it flagged and told them where I got them from.  No idea if that'll help in the future, but hope so.
